app/routes/_index.jsx

```diff
@@ -2,38 +2,62 @@ import { json } from "@remix-run/node";
 import { useLoaderData } from "@remix-run/react";
 import styles from "../styles/index.css?url";
 import MovieCard, { links as movieCardLinks } from "../components/MovieCard/MovieCard";
+import Pagination, { links as paginationLinks } from "../components/Pagination/Pagination";
 
 export const meta = () => [
   { title: "New Remix App" },
   { name: "description", content: "Welcome to Remix!" },
 ];
 
-export const loader = async () => {
-  const res = await fetch("https://freetestapi.com/api/v1/movies?limit=20");
-  return json(await res.json());
+export const loader = async ({ request }) => {
+  try {
+    const url = new URL(request.url);
+    const page = url.searchParams.get("page");
+    const res = await fetch(
+      `http://www.omdbapi.com/?s=movie&apikey=5ccd3d05&page=${page}`
```

Comment:
move api key to environment variable
move base url path to variable and re-use it everywhere to avoid misprints